src/main/java/trinity/play2learn/backend/user/services/jwt/JwtService.java

```diff
@@ -72,6 +71,11 @@ private Claims extractAllClaims(String token) {
                 .getBody();
     }
 
+    //Setea el tiempo de expiracion
+    private long getExpirationTime() {
+        return 1000 * 60 * 60 * 24; // Token expira en 24 horas
+    }
```

Comment:
Movelo a una constante que tenga el resultado de la expiración (o bien el cálculo). Llamar al método es más costoso que llamar a una constante.

```suggestion
    private static final long EXPIRATION_TIME = 86400000;
```